### PR TITLE
Update fiction.md

### PR DESCRIPTION
**Pull request by ProjitB**
Abstract changes.

### CODE REVIEW

File: fiction.md [hunk 1]
```diff
@@ -31,22 +31,22 @@ of matching parity.
...
-gates. This permits a curious form of self-modifying
-programming that allows access to better complexity classes.
-We propose three new lines of research: one on uncovering the
-complexity theoretic phenomena that is present, another
-on the symplectic geometry that is controlling the situation,
-and finally, the intruguing possibility of building such a machine.
+of this model comes from allowing rearrangements of the billiard
+ball evolution. The balls act as inputs, outputs, and
+gates. As a result, a curious form of self-modifying
+programming which allows access to better complexity classes, is formed.
```
Comment (bollu) [inline fiction.md:45]: the hanging `,` of `, is formed` feels very bad to me.

Comment (ProjitB) [reply fiction.md:45]: Ah yes. At its core.. "A curious form of self-modifying programming, which allows access to better complexity classes, is formed." is the sentence that it should be.
The part between the commas modifies the noun preceding it, and it should be removable by itself with the sentence making sense. There should be another comma after programming essentially. The "As a result" is stylistic.
The sentence itself needed to be changed as it contained "This permits...", ... "allows.."  in the same sentence. So wanted to get rid of at least one.

Comment (bollu) [reply fiction.md:45]: I would prefer the above sentence to be:

> a curious form of self-modifying
programming which allows access to better complexity classes is formed.

(eliminate the `,`)

Comment (ProjitB) [reply fiction.md:45]: Not 100% sure that is right, without the `,`.
It reads fine though without it..but requires the reader to pause correctly.

Comment (bollu) [reply fiction.md:45]: The existence of the `,` feels really awkward. I'm unable to explain why.

Comment (ProjitB) [reply fiction.md]: Sure, changed it to `()`

Comment (ProjitB) [reply fiction.md]: > The existence of the `,` feels really awkward. I'm unable to explain why.

Probably due to too many commas in the sentence with it.

**Pull request merged**